### ISSUE TITLE
Gutenboarding: Verification email shouldn't include the username

### ISSUE THREAD

**Issue by johnHackworth** [Automattic/wp-calypso#41307]
Right now, when a user register with gutenboarding, we are auto-generating a username for them. The problem is they don't see that username until they get the email to verify their new WordPress.com account, which includes the random user name in the subject.
Let's remove the username from the subject if an account has been created on gutenboarding, and go for a more generic email subject

**Comment by ramonjd** [comment]
We can look at this. I think we've done something similar.

Some related discussion in a Slack thread noted over at https://github.com/Automattic/wp-calypso/issues/38796#issuecomment-609485307

**Comment by p-jackson** [comment]
I'm thinking I'll make this change for all passwordless signups, not just for gutenboarding.
- It seems like any user would be confused by this, not just gutenboarding users
- Mobile (who also use passwordless signup) already have their own emails so won't impact them
- Our other passwordless signups are behind an a/b test that's receiving 0 users. So we won't be suprising anyone there either.

**Comment by p-jackson** [comment]
Looks like we've already got code to do this for passwordless accounts. Plus the passwordless email templates are better for this situation too e.g. talks about the username in the email body, but worded in such that it's clear that this is something we've picked for them.

I'm going to copy these templates and just remove the paragraphs about remembering to set a password. Plus side is the text will already be translated 👍
